scripts/build/index.test.ts

```diff
@@ -0,0 +1,85 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import assert from 'node:assert/strict';
+
+import { walk } from '../../utils/index.js';
+
+import { generateMeta, applyMirroring, addVersionLast } from './index.js';
```

Comment:
Lets test manifest creating as well.

```suggestion
import {
  generateMeta,
  applyMirroring,
  addVersionLast,
  createManifest,
} from './index.js';
```